packages/proxy/src/logger.ts

```diff
@@ -0,0 +1,21 @@
+import * as winston from "winston";
+
+export const Logger = (function () {
+  let instance: winston.Logger;
+
+  return {
+    getLogger: function () {
```

Comment:
I like Java too but chill out bro

Reply:
Nah you got me confused for someone else. I can do without Java, I just wanted a singleton.

Reply:
just name it in all CAPS and then everyone knows it's a singleboi, MECHANISMS

Reply:
actually you know what, don't. Just write about it in a wiki

Reply:
STOP SAYING REVISION THIS IS GITHUB JUST PUSH A COMIT BIG DAWG

Reply:
You're right. I will make a new branch and cut a new PR.